src/views/Studies/components/PageContent/index.tsx

```diff
@@ -0,0 +1,154 @@
+import ProTable from '@ferlab/ui/core/components/ProTable';
+import GridCard from '@ferlab/ui/core/view/v2/GridCard';
+import { UserOutlined } from '@ant-design/icons';
+import { Space, Typography } from 'antd';
+import { getProTableDictionary } from 'utils/translation';
+import intl from 'react-intl-universal';
+import { isEmptySqon, resolveSyntheticSqon } from '@ferlab/ui/core/data/sqon/utils';
+import { combineExtendedMappings } from 'utils/fieldMapper';
+
+import styles from './index.module.scss';
+import { ProColumnType } from '@ferlab/ui/core/components/ProTable/types';
+import QueryBuilder from '@ferlab/ui/core/components/QueryBuilder';
+import { getQueryBuilderDictionary } from 'utils/translation';
+import {
+  DEFAULT_PAGE_INDEX,
+  DEFAULT_PAGE_SIZE,
+  DEFAULT_QUERY_CONFIG,
+  SCROLL_WRAPPER_ID,
+  STUDIES_REPO_QB_ID,
+} from '../../utils/constant';
+import { ExtendedMapping, ExtendedMappingResults } from 'graphql/models';
+import { useEffect, useState } from 'react';
+import useQueryBuilderState from '@ferlab/ui/core/components/QueryBuilder/utils/useQueryBuilderState';
+import { useStudies } from 'graphql/studies/actions';
+import { formatQuerySortList, scrollToTop } from 'utils/helper';
+import { useDispatch } from 'react-redux';
+import { fetchTsvReport } from 'store/report/thunks';
+import { INDEXES } from 'graphql/constants';
+import { useUser } from 'store/user';
+
+const { Title } = Typography;
+
+type OwnProps = {
+  defaultColumns: ProColumnType<any>[];
+  extendedMappingResults?: ExtendedMappingResults;
+};
+
+const PageContent = ({
+  defaultColumns = [],
+  extendedMappingResults = { data: [], loading: false },
+}: OwnProps) => {
+  const dispatch = useDispatch();
+  const { userInfo } = useUser();
+  const { queryList, activeQuery } = useQueryBuilderState(STUDIES_REPO_QB_ID);
+  const [queryConfig, setQueryConfig] = useState(DEFAULT_QUERY_CONFIG);
+  const resolvedSqon = resolveSyntheticSqon(queryList, activeQuery);
+
+  const { loading, total, data } = useStudies({
+    first: queryConfig.size,
+    offset: queryConfig.size * (queryConfig.pageIndex - 1),
+    sqon: resolvedSqon,
+  });
+
+  useEffect(() => {
+    setQueryConfig({
+      ...queryConfig,
+      pageIndex: DEFAULT_PAGE_INDEX,
+    });
+    // eslint-disable-next-line
+  }, [JSON.stringify(activeQuery)]);
```

Comment:
is there a particular field that we could check (an id for instance). JSON.stringnify can lead to subtle bugs.